qiskit_experiments/database_service/db_experiment_data.py

```diff
@@ -745,7 +753,7 @@ def save(self) -> None:
             )
             return
 
-        self.save_metadata()
+        DbExperimentDataV1.save_metadata(self)
```

Comment:
Should this still be self?

Reply:
I don't understand the question

Reply:
Why is it changed from `self.save_metadata()`? This is quite hacky since you are basically treating regular method `save_metadata` like a `staticmethod`.